tests/Microsoft.AspNet.SignalR.Tests/packages.config

```diff
@@ -1,7 +1,12 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Moq" version="4.0.10827" />
+  <package id="Moq" version="4.0.10827" requireReinstallation="true" />

```

Comment:
Yeah, this is intended to be a flag for the UI to prompt you to reinstall the package, at least that's what I thought it was.